public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx

```diff
@@ -128,9 +128,9 @@ export class PromQueryEditor extends PureComponent<Props, State> {
           <div className="gf-form">
             <FormLabel
               width={7}
-              tooltip="Leave blank for auto handling based on time range and panel width.
-            Note that the actual dates used in the query will be adjusted
-        to a multiple of the interval step."
+              tooltip="An additional lower limit for the step parameter
+        of the Prometheus query and for the <code>$__interval</code> variable.
```

Comment:
@davkal Is it OK to have markup tags in this string?

Reply:
Fixed that for you 🎄